packages/react-components/src/components/MeetingReactionOverlay.tsx

```diff
@@ -125,6 +139,28 @@ export const MeetingReactionOverlay = (props: MeetingReactionOverlayProps): JSX.
         />
       </div>
     );
+  } else if (props.overlayMode === 'together-mode') {
+    /* @conditional-compile-remove(together-mode) */
+    return (
+      <div
+        style={{
```

Comment:
Can we have this style in a style file? 
Looks like there is quite a bit style applied here.